src/main/java/net/minecraftforge/event/datafix/RegisterFixesEvent.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2022.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.event.datafix;
+
+import com.google.common.collect.Lists;
+import com.mojang.datafixers.DSL;
+import com.mojang.datafixers.DataFix;
+import com.mojang.datafixers.DataFixer;
+import com.mojang.datafixers.schemas.Schema;
+import com.mojang.datafixers.types.Func;
+import com.mojang.datafixers.types.templates.TypeTemplate;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.util.datafix.fixes.References;
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraftforge.fml.ModContainer;
+import net.minecraftforge.fml.event.IModBusEvent;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+/**
+ * Event fired in parallel on the mod event bus to allow for registration of new fixers to a given schema.
+ */
+public class RegisterFixesEvent extends Event implements IModBusEvent
+{
+    private final Schema schema;
+    private final ModContainer                      container;
+    private final List<Function<Schema, DataFix>> fixerFactories;
+
+    public RegisterFixesEvent(
+      final Schema schema,
+      final ModContainer container
+    ) {
```

Comment:
Formatting: braces on newlines.
```suggestion
    )
    {
```